mteb/tasks/BitextMining/multilingual/DiaBLaBitextMining.py

```diff
@@ -42,10 +44,8 @@ def load_data(self, **kwargs):
 
         self.dataset = {}
         for lang in self.langs:
-            self.dataset[lang] = datasets.load_dataset(
-                self.metadata_dict["hf_hub_name"],
-                revision=self.metadata_dict.get("revision", None),
-            )
+            # TODO it looks like the lang is not used here, should we pass `name=lang`?
+            self.dataset[lang] = datasets.load_dataset(**self.metadata_dict["dataset"])
```

Comment:
It looks like the lang is not used here, should we pass `name=lang`?

Reply:
@imenelydiaker?

Reply:
Actually, there is only one split in [DiaBLa](https://huggingface.co/datasets/rbawden/DiaBLa), so even if you specify the language it won't change anything (it may cause an error). 
In the first implementation the language handled was only `"fr-en"`, and that's what the `data_transform()` function builds (first column with French values and second with English values). 
I see that "en-fr" was added to the metadata, but it is exactly the same dataset, I'm not sure it is relevant.